File: 단계별로_풀어보기/2. 조건문/5. 2884.py
import sys, math

input = sys.stdin.readline
output = sys.stdout.write

H, M = map(int, input().split())

HM = 60 * H + M + 1440 - 45

resultH = (HM // 60) % 24
resultM = HM % 60

output(f"{resultH} {resultM}")

File: 1. 단계별로_풀어보기/1~9/2. 조건문/5. 2884.js
const fs = require('fs')
const inputs = fs.readFileSync('/dev/stdin').toString().split(' ')

const [h, m] = inputs

const [H, M] = [parseInt(h), parseInt(m)]

const sumHM = 60 * H + M + 1440
const resultHM = sumHM - 45

const resultH = Math.floor(resultHM / 60) % 24
const resultM = resultHM % 60

process.stdout.write(resultH.toString() + ' ' + resultM.toString())
